ibtk/config/config.h.tmp

```diff
@@ -62,6 +62,8 @@ _Pragma("GCC diagnostic ignored \"-Wunneeded-internal-declaration\"")
 
 #endif // #ifdef IBTK_HAVE_PRAGMA_KEYWORD
 
+#include <ibtk/samrai_interface.h>
```

Comment:
Could we name this `samrai_compatibility` instead?

Reply:
Sure.